src/calculateRentalCost.js

```diff
@@ -28,7 +28,21 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const rent = 40;
+
+  if (days >= 7) {
+    return (days * rent) - 50;
```

Comment:
-50 and - 20 should be separate variables, because it is not clear from the context what they mean